src/main/java/nl/tudelft/contextproject/gui/PreviewController.java

```diff
@@ -58,13 +62,18 @@
     @FXML private TextField duration;
 
     private SimpleIntegerProperty currentShot;
-    private int currentSelected;
+    private Shot currentFirst;
     private Timeline timeline;
     private List<Shot> shots;
     private List<ImageView> views;
+    private Script script;
 
     @FXML private void initialize() {
-        currentSelected = 1;
+        // backend
+//        script = ContextTFP.getScript();
+//        shots = script.getShots();
+//        actualCam = shots.get(0).getCamera();
```

Comment:
The actualCam should be script.getCurrentShot.getCamera() not shots.get(0).getCamera()